docs/sources/datasources/alertmanager.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "Alertmanager"
+description = "Guide for using Alertmanager in Grafana"
+keywords = ["grafana", "prometheus", "guide"]
+aliases = ["/docs/grafana/latest/features/datasources/alertmanager"]
+weight = 1300
++++
+
+# Alertmanager data source
+
+Grafana includes built-in support for Prometheus Alertmanager. Add it as a datasource and you will be able to use the [Grafana alerting UI](https://grafana.com/docs/grafana/latest/alerting/) to manage silences, contact points and notification policies. A dropdown will appear in these pages to allow you to switch between Grafana and any configured alertmanager data sources This data source cannot return graph data and is not available in panels.
+Refer to [Add a data source]({{< relref "add-a-data-source.md" >}}) for instructions on how to add a data source to Grafana. Only users with the organization admin role can add data sources.
+
+>**Note:** New in Grafana 8.0.
+
+>**Note:** Currently only [Cortex Alertmanager](https://cortexmetrics.io/docs/proposals/scalable-alertmanager/) is supported.
```

Comment:
Is this Cortex Alertmanager or Prometheus?

Reply:
both in a way, as in Cortex implementation of Prometheus Alertmanager :sweat_smile:

Reply:
Can we make this little clear, then we should be good to go. Thanks!

Reply:
Updated to "Cortex implementation of Prometheus Alertmanager"